### PR TITLE
Feature/additional validators

### PR DESCRIPTION
**Pull request by SergeiMikhailovskii**
Added validators (#44)

### CODE REVIEW

File: form-builder/src/test/java/com/dsc/form_builder/TextFieldProviders.kt [hunk 1]
```diff
@@ -13,6 +13,34 @@ object EmailArgumentsProvider : ArgumentsProvider {
...
+    override fun provideArguments(context: ExtensionContext?): Stream<out Arguments> = Stream.of(
+        Arguments.of("test", false),
+        Arguments.of("88005553535", true),
+        Arguments.of("+8(800) 555-35-35", true),
+        Arguments.of("8-800-555-35-35-", false),
+    )
+}
+
+object WebUrlArgumentsProvider : ArgumentsProvider {
+    override fun provideArguments(context: ExtensionContext?): Stream<out Arguments> = Stream.of(
```
Comment (LinusMuema) [inline form-builder/src/test/java/com/dsc/form_builder/TextFieldProviders.kt:26]: Could you add a test for "www.test.com" It's a valid web url and should be factored in. The test is failing once we add this.

**Review by LinusMuema — COMMENTED** [review]
Hi @SergeiMikhailovskii 😃

Thanks for contributing to our library and we appreciate it. 

I have reviewed your work and there's only one suggestion below. You can look into it then we can merge the pull request.

File: form-builder/src/test/java/com/dsc/form_builder/TextFieldProviders.kt (hunk 1 above)
Comment (SergeiMikhailovskii) [reply form-builder/src/test/java/com/dsc/form_builder/TextFieldProviders.kt:26]: thanks for the review! fixed the validator in the new commit

**Review by LinusMuema — APPROVED** [review]
@SergeiMikhailovskii 

Thanks a lot for your contributions and helping make this library great.

**Pull request merged**